mava/utils/jumanji_registry.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2022 InstaDeep Ltd. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, Callable, Dict
+
+# Define the ranges of valid attributes
+LBF_ATTRIBUTES_RANGES: Dict[str, Any] = {
+    "grid_size": range(5, 20),  # Grid size, the height and width are equal.
+    "fov": range(2, 20),  # Field of view of agent.
+    "num_agents": range(2, 20),
+    "num_food": range(1, 10),
+    "force_coop": [True, False],  # Force cooperation between agents.
+}
+
+RWARE_ATTRIBUTES_RANGES: Dict[str, Any] = {
+    "num_agents": range(1, 20),
+    "difficulties": {"-easy": 2, "": 1, "-hard": 0.5},
+    "size": {
+        "tiny": (1, 3),
+        "small": (2, 3),
+        "medium": (2, 5),
+        "large": (3, 5),
+    },
+}
+
+
+def get_task_config(env_name: str, scenario: str) -> Any:
+    """
+    Retrieves the attributes for a given environment and scenario.
+
+    Args:
+        env_name (str): The name of the environment.
+        scenario (str): The scenario to retrieve attributes for.
+
+    Returns:
+        dict: A dictionary containing the attributes of the scenario.
+    """
+    if env_name not in ENV_FUNCTIONS:
+        raise ValueError(f"Unknown environment name: {env_name}")
```

Comment:
Nice to have the available names also :+1: 

```suggestion
        raise ValueError(f"Unknown environment: {env_name}. Available environments: {' '.join(ENV_FUNCTIONS.keys()}")
```